integreat_cms/cms/forms/regions/region_form.py

```diff
@@ -332,15 +330,14 @@ def save(self, commit: bool = True) -> Region:
                         region,
                     )
                     duplicate_imprint(source_region, region)
-            pre_save.connect(
-                receiver=page_translation_save_handler, sender=PageTranslation
-            )
             # Duplicate media content
             duplicate_media(source_region, region)
 
             # Create links for the most recent versions of all translations manually and replace internal links
             create_and_replace_links_async(source_region, region)
 
+        if hix_enabled:
+            region.hix_enabled = True
```

Comment:
I feel like we can easily just use it directly here. We're not using the variable elsewhere anyway
```suggestion
        region.hix_enabled = self.cleaned_data["hix_enabled"]
```